rpcclient/infrastructure.go

```diff
@@ -1545,7 +1545,7 @@ const (
 
 	// bitcoindVersionPrefix specifies the prefix included in every bitcoind
 	// version exposed through GetNetworkInfo.
-	bitcoindVersionPrefix = "/Satoshi:"
+	bitcoindVersionPrefix = "/Dash Core:"
```

Comment:
1. Do we really need to have a space?
2. Could you update doc block and probably the name of a constant?